assets/multiqc_config.yaml

```diff
@@ -59,6 +59,8 @@ extra_fn_clean_exts:
   - ".trimmed_stats"
   - "_libmerged"
   - "_bt2"
+  - type: "regex"
+    pattern: "_udg(half|none|full)"
```

Comment:
Missing changelog

Reply:
It is implicitly there. I can make it clearer?

Reply:
Nope, I just can't read - all good!